sound/soc/sof/Makefile

```diff
@@ -14,13 +14,9 @@ snd-sof-dma-trace-objs := sof-client-dma-trace.o
 
 snd-sof-nocodec-objs := nocodec.o
 
-snd-sof-utils-objs := sof-utils.o
-
 obj-$(CONFIG_SND_SOC_SOF) += snd-sof.o
 obj-$(CONFIG_SND_SOC_SOF_NOCODEC) += snd-sof-nocodec.o
 
-obj-$(CONFIG_SND_SOC_SOF) += snd-sof-utils.o
```

Comment:
why do you actually want to have it as a separate module? Why not keep it in the core module?

Reply:
I'm not sure to be honest. Are you fine with the split up otherwise? I have a followup patch to compile it in, I can just squash that in.